src/component/user/HostedAuth/HostedAuth.tsx

```diff
@@ -52,8 +59,11 @@ const HostedAuth = ({ authDetails, redirect }) => {
             await passwordAuth(authDetails.path, username, password);
             refetchUser();
             history.push(redirect);
-        } catch (error) {
-            if (error.statusCode === 404 || error.statusCode === 400) {
+        } catch (error: any) {
+            if (
+                error?.statusCode === NOT_FOUND ||
```

Comment:
Maybe something like this could work.

```
} catch (error: unknonw) {
    if (error instanceof NotFoundError || error instanceoof BadRequestError) {
    	/// ...
```